### PR TITLE
Fix dictionary encoding issues in non english languages

### PR DESCRIPTION
**Pull request by mkortkamp**
- this fix corrects the broken encoding in non english languages (see sebastian-lempert's recent mail on the mucommander mailing list)
- the cause was that with one of the last changes before spliting the dictionary.txt the old single file was saved in non UTF-8
- all dictionaries are restored as UTF-8 from this old version and all changes since then are applied again
- Translator.java is changed to load these files as UTF-8
- gradle eclipse import hook was added to automatically set the correct encoding of these files

### CODE REVIEW

**Comment by mkortkamp** [comment]
fixes Issue #21

File: src/main/resources/dictionary_ZH_CN.properties [hunk 1]
```diff
@@ -1,8 +0,0 @@
-#pack.error_on_file = 压缩时发生错误 {0}
-#pack_dialog.cannot_write = 无法在目的文件夹创建压缩文件
-#unpack.unable_to_open_zip = 无法打开压缩文件 {0}
-#mkdir_dialog.title = 新建文件夹
-#edit_bookmarks_dialog.remove = 删除
-#mkdir_dialog.description = 创建文件夹
-#done = 完成
-#progress_dialog.hide = 隐藏
```
Comment (ahadas) [inline src/main/resources/dictionary_ZH_CN.properties]: s/unsed/unused in the commit message

**Comment by ahadas** [comment]
very nice contribution!
please just fix the minor comment inside and I will merge it

File: src/main/resources/dictionary_ZH_CN.properties (hunk 1 above)
Comment (mkortkamp) [inline src/main/resources/dictionary_ZH_CN.properties]: done

**Comment by mkortkamp** [comment]
comment fixed

**Pull request merged**